### DISCUSSION TITLE
Quick Reference Guide & case sensitive

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1725] [Q&A]
If vd shortcuts are case sensitive, then why is SHIFT included "Columns Sheet (Shift+C)" but not other commands in the quick reference? (I can understand why it is specified on arrow commands...)

**Comment by saulpw** [comment]
I'd like to move to a consistent `Shift+X` (and `Ctrl+X` and `Alt+X`) format for keystrokes for all of our documentation, and indeed even the keybinding strings themselves.  The manpage is several years old at this point and is not so easy to update, so it could use a reworking; probably at that point we'll convert them all to the same format.

**Comment by reagle** [comment]
As an aside, perhaps there could be a date/version on the [Quick Reference Page](https://www.visidata.org/docs/man/) so users know how out of date it might be? Could such documentation be brought over to github for easier management?

**Reply by saulpw** [reply]
It's right there in the main visidata repo: https://github.com/saulpw/visidata/blob/develop/visidata/man/vd.inc

I'm not sure why it's not being updated on visidata.org/man for our releases; that's a bug.  But like I said, the whole thing could use an overhaul anyway so that it's easier to maintain.
